README.md

```diff
@@ -5,7 +5,7 @@ Usermode Driver for Tenstorrent AI Accelerators
 ## Dependencies
 Required Ubuntu dependencies:
 ```
-sudo apt install -y libhwloc-dev cmake ninja-build
+sudo apt install -y cmake ninja-build libnuma-dev
```

Comment:
You do have to add this to build_image.yaml, and run Build Docker workflow on this branch to build another image which will have this lib, right?